R/gtsave.R

```diff
@@ -78,22 +93,58 @@ gt_save_html <- function(data, filename, ..., inline_css = FALSE) {
   }
 }
 
-# Saving function for a LaTeX file
+#' Saving function for a LaTeX file
+#'
+#' @importFrom htmltools HTML save_html
+#' @noRd
+gt_save_webshot <- function(data, filename, ...) {
+
+  tempfile_ <- tempfile(fileext = ".html")
+
+  data %>%
+    as_raw_html(inline_css = TRUE) %>%
+    htmltools::HTML() %>%
+    htmltools::save_html(file = tempfile_)
+
+  if (requireNamespace("webshot", quietly = TRUE)) {
+
+    # Save the image in the working directory
+    webshot::webshot(
+      url = paste0("file://", tempfile_),
```

Comment:
Can you make sure to try this on Windows? I'd want to make sure it works fine with Windows-style paths (i.e. starting with a drive letter) and also with paths that have spaces (which I think Windows temp paths do, by default).

Reply:
I will test this on Windows for sure.